products/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+class Category(models.Model):
+    name       = models.CharField(max_length=100)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'categories'
+
+class Product(models.Model):
+    category      = models.ForeignKey('Category', on_delete=models.CASCADE)
+    name          = models.CharField(max_length=100)
+    origin        = models.CharField(max_length=100, default='국산')
+    weight_volume = models.IntegerField(max_length=100)
+    description   = models.TextField()
+    price         = models.IntegerField()
+    created_at    = models.DateTimeField(auto_now_add=True)
+    updated_at    = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        db_table = 'products'
+
+class Picture(models.Model):
+    product    = models.ForeignKey('Product', on_delete=models.CASCADE)
+    img        = models.CharField(max_length=1000)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'pictures'
```

Comment:
`img`  -> `image_url` 혹은 `url`로 수정해주세요. 실제로 컬럼에 저장되는 값은 image_url 이기 때문에 컬럼 명도 명확하게 만들어주셔야 합니다.
```suggestion
class Picture(models.Model):
    product    = models.ForeignKey('Product', on_delete=models.CASCADE)
    url        = models.CharField(max_length=1000)
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)

    class Meta:
        db_table = 'pictures'
```